src/prev7Mappings/mapping.ts

```diff
@@ -33,6 +33,24 @@ export function handleTransfer(event: Transfer): void {
   }
 }
 
+export function handleCancelKey(event: CancelKey): void {
```

Comment:
I think this below is not a real change? (just moved aroud in the file, correct?)

Reply:
The is a rename of the previously existing file and the exported methods have been placed at the top of the file